_includes/feature_row

```diff
@@ -16,8 +16,8 @@
         src="{{ f.image_path | relative_url }}"
         alt="{% if f.alt %}{{ f.alt }}{% endif %}">
     </div>
-    <div style="display:inline-block; width:230px; line-height:1.4;">
-    <p style="margin:0 0 0 10px;">{% if f.person_name %}{{ f.person_name | markdownify | remove: "<p>" | remove: "</p>" }}{% endif %}</p>
+    <div style="display:inline-block; width:230px; line-height:1.4; vertical-align: top">
```

Comment:
@yrlu we also want to do that for lines 35,36, right? (In case there are two speakers for a slot, as currently for the Waymo and Waabi slots)

Reply:
Sure, updated line 35, 36.